src/main/resources/application.properties

```diff
@@ -7,14 +7,15 @@ quarkus.http.port=8085
 %test.quarkus.http.port=9085
 %test.quarkus.http.test-port=9085
 
+%prod.quarkus.health.extensions.enabled=false # TODO [BG Phase 2.1] Remove this line
```

Comment:
We would still show as `UP` in the live check?

Reply:
I'm not sure what impact it will have on OpenShift. On my machine it removes all DOWN health checks. Only that one remains:

```
{
    "status": "UP",
    "checks": [
        {
            "name": "Notifications readiness check",
            "status": "UP",
            "data": {
                "reactive-db-check": true
            }
        }
    ]
}
```